packages/contracts/migrations/development/Deploy001_Diamond_Dollar.s.sol

```diff
@@ -332,4 +353,114 @@ contract Deploy001_Diamond_Dollar is Script, DiamondTestHelper {
         // stop sending admin transactions
         vm.stopBroadcast();
     }
+
+    /**
+     * @notice Initializes oracle related contracts
+     *
+     * @dev Ubiquity protocol supports 2 oracles:
+     * 1. Curve's Dollar-3CRVLP metapool to fetch Dollar prices
+     * 2. Chainlink's price feed (used in UbiquityPool) to fetch collateral token prices in USD
+     *
+     * There are 2 migrations (deployment scripts):
+     * 1. Development (for usage in testnet and local anvil instance forked from mainnet)
+     * 2. Mainnet (for production usage in mainnet)
+     *
+     * Development migration deploys (for ease of debugging) mocks of:
+     * - Chainlink price feed contract
+     * - 3CRVLP ERC20 token
+     * - Curve's Dollar-3CRVLP metapool contract
+     */
+    function initOracles() public virtual {
+        //========================================
+        // Chainlink LUSD/USD price feed deploy
+        //========================================
+
+        // start sending owner transactions
+        vm.startBroadcast(ownerPrivateKey);
+
+        // deploy LUSD/USD chainlink mock price feed
+        chainLinkPriceFeedLusdUsd = new MockChainLinkFeed();
+
+        // stop sending owner transactions
+        vm.stopBroadcast();
+
+        //=======================================
+        // Chainlink LUSD/USD price feed setup
+        //=======================================
+
+        // start sending admin transactions
+        vm.startBroadcast(adminPrivateKey);
+
+        // set params for LUSD/USD chainlink price feed mock
+        MockChainLinkFeed(address(chainLinkPriceFeedLusdUsd)).updateMockParams(
+            1, // round id
+            100_000_000, // answer, 100_000_000 = $1.00 (chainlink 8 decimals answer is converted to 6 decimals used in UbiquityPool)
+            block.timestamp, // started at
+            block.timestamp, // updated at
+            1 // answered in round
+        );
+
+        UbiquityPoolFacet ubiquityPoolFacet = UbiquityPoolFacet(
+            address(diamond)
+        );
+
+        // set price feed address and set threshold to 10 years (3650 days) for ease of debugging
+        ubiquityPoolFacet.setCollateralChainLinkPriceFeed(
+            collateralTokenAddress, // collateral token address
+            address(chainLinkPriceFeedLusdUsd), // price feed address
+            3650 days // price feed staleness threshold in seconds
```

Comment:
let's add this to a constant state var

Reply:
fixed https://github.com/ubiquity/ubiquity-dollar/pull/855/commits/24dbf9f7643fd7a8f4fd96788714c2fdf9055984

P.S. Constant state vars can't be overridden hence `CHAINLINK_PRICE_FEED_THRESHOLD` is a plain state var